code/topology/OOTB_install.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+OCFILE="../../blogs/scripted-windows/oc-creds.yaml"
+YAMLFILE="../../blogs/scripted-windows/"
+
+# Check if the bak file exists and exit the script
+#in order to make sure that nothing is overwritten
+
+if [ -f "$OCFILE.bak" ]; then
+	echo "config file exists in $OCFILE.bak"
+        exit 1
+fi
+
+
+# Sets the connection to the OpenShift cluster
+echo "Please enter OpenShift login details:"
+
+echo "OpenShift API:"
+read OCAPI
+
+echo "OpenShift Username:"
+read OCUSER
+
+echo "OpenShift Pass:"
+read OCPASS
+
+# Encode the values to fit the yaml 
+API=$(echo "$OCAPI" | base64)
+USER=$(echo "$OCUSER" | base64)
+PASS=$(echo "$OCPASS" | base64)
+
+sed -i.bak "s/<base64 URL>/$API/" $OCFILE
+sed -i.bak "s/<base64 user>/$USER/" $OCFILE
+sed -i.bak "s/<base64 password>/$PASS/" $OCFILE
+
```

Comment:
You go through all the trouble of making the credential grab interactive, but you never create the secret in the cluster from the file...

Reply:
Talked, Fixed the Bak file